### DISCUSSION TITLE
Understanding the step response pastas

### DISCUSSION THREAD

**Discussion by martafaneca** [pastas/pastas#706] [Q&A] [answered]
Could you help interpret the step-reponse graphics generated with pastas? And is it possible to obtain the value of the step-response? 

![Capture](https://github.com/pastas/pastas/assets/10071908/18dcf8a6-7940-44e2-913a-1db114d8b765)

I cannot figure out how to interpret it, but if I look at the paper https://ngwa.onlinelibrary.wiley.com/doi/full/10.1111/gwat.12925 (explanation after figure 5) then it seems I could interpret it like this:

In the case below, for the recharge, can we say that if there is 1mm/d of recharge continuously the groundwater levels will reach a head of aprox 1.5m (or is it a change?) after 2000 days? or is it 3000 days or more? and for the wells, can we say that if we pump 1m3/d all the time, the head will drop 0.01m after aprox 100 days? 
but I am not sure this is correct.

could you help?

Thank you very much!

**Comment by FransSchaars** [comment, marked as answer]
Hi @martafaneca,
Thank you for your question. 
The response function is the basic building block of time series analysis. 
You might want to read this article: [Solving Groundwater Flow Problems with Time Series Analysis: You May Not Even Need Another Model]( https://ngwa.onlinelibrary.wiley.com/doi/full/10.1111/gwat.12927). 
Your interpretation is right, but the step respons is change in head caused by a unit change in the stress. 
You can find the final change (called gain) denoted by parameter `_A` in the parameter list (`model.parameters`) in the `model.plots.results()` figure. To get the entire step response use `model.get_step_response()`. See for instance the example Notebook: https://pastas.readthedocs.io/en/master/examples/changing_responses.html.

**Reply by martafaneca** [reply]
Thank you very much for the swift answer!
